Grammar Updates to the General Folder (#1026)
AdguardTeam/AdguardKnowledgeBase: 01.general/08.how-malware-protection-works/docs.en.md

## Introduction

Every day, tens of thousands of people fall victim to virus attacks or fraud on the Internet. To protect AdGuard users from such threats, we have added special filters to the program that protect you from malicious and phishing websites.

> At the moment, we have categorized more than 15 million sites, and our filters contain about 1.5 million phishing and malware sites. Just think about it: 10% of all known sites can be dangerous for you! Inexperienced users can easily become victims of fraudsters or hackers. We hope that the use of protection against malicious websites will help you avoid all hazards that you may encounter.

Author: ```suggestion
So far, we have categorized more than 15 million sites, and our filters contain about 1.5 million phishing and malware sites. Just think about it: 10% of all known sites can be dangerous for you! Inexperienced users can easily become victims of fraudsters or hackers. We hope that using protection against malicious websites will help you avoid all the hazards that you may encounter.
```
